docs/install.md

```
@@ -140,9 +147,17 @@ kubectl get pods -n skyflo-ai
-Your services are directly accessible through NodePorts:
-- UI: http://localhost:30080
-- API: http://localhost:30081
+Use port forwarding to expose the UI pod to your local machine:
+
+```bash
+kubectl port-forward <ui-pod-name> [LOCAL PORT]:[REMOTE PORT]
+```
+Replace `<ui-pod-name>`, `[LOCAL PORT]`, and `[REMOTE PORT]` as needed (e.g., `kubectl port-forward skyflo-ai-ui-xxxx 30080:80`).
+
+Then access your local installation at:
+- UI: http://localhost:[LOCAL PORT]
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)